### DISCUSSION TITLE
Password management

### DISCUSSION THREAD

**Discussion by egolfeld** [isontheline/pro.webssh.net#480] [Ideas]
If user don't want to save password, when user can fill it from password manager app, like kypass, but  ionly on connection time, when Webssh asks for password. 

If you can add good way to past it on remote host (for sudo, for example) without manual access to kypass (to copy to clipboard), when it make webssh the best ssh.

Maybe some checkbox on password prompt modal to store in memory it for a session, or for X hours, and a button to past it?

Or maybe somehow add ability to use kypass in the ssh directly?

Like this:
![9304CD90-9B82-463B-9F54-DD60B1747534](https://user-images.githubusercontent.com/35452983/146832928-5381dc50-cd84-4b9a-8ca0-6307ab25e80a.png)

**Comment by isontheline** [comment]
Good one 👍
Issue created : https://github.com/isontheline/pro.webssh.net/issues/485
